client/src/pages/Login.js

```diff
@@ -21,9 +19,21 @@ export default function Login() {
       password,
     };
 
-    loginUser(newUser);
-    dispatch({ type: SET_AUTHENTICATED });
-    history.push("/");
+    loginUser(newUser)
+      .then((user) => {
+        dispatch({
+          type: SET_USER,
+          payload: {
+            user: {
```

Comment:
```suggestion
            user: { name, image, email }
```

just small thing for shorthand :)